docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -80,6 +80,36 @@ At a minimum, ``hive.metastore.uri`` must be configured:
     - Maximum number of partitions handled per writer.
     - 100
 
+.. _iceberg-authorization:
+
+Authorization checks
+^^^^^^^^^^^^^^^^^^^^
+
+You can enable authorization checks for the connector by setting
+the ``iceberg.security`` property in the catalog properties file. This
```

Comment:
Is the 'iceberg.security' property in the catalog properties file by default or does the user need to add it and set the values?

If they have to add it to the file and choose a value, I suggest reframing the sentence to something like:

> To enable authorization checks, add the 'iceberg.security' property to the connector catalog properties file.

Reply:
I don't disagree, but we've used the wording "set/by setting the `example.property` property in the catalog properties file" just about everywhere else in the connector docs. This property is no different, I'd leave it as-is.